Realm/RLMObjectBase.h

```diff
@@ -23,8 +23,6 @@
 @class RLMObjectSchema;
 
 @interface RLMObjectBase : NSObject
-@property (nonatomic, readonly) RLMRealm *realm;
-@property (nonatomic, readonly) RLMObjectSchema *objectSchema;
 @property (nonatomic, readonly, getter = isInvalidated) BOOL invalidated;
```

Comment:
If at all possible, we should move `initWithObject:`, `initWithObjectSchema:`, `isEqualToObject:` and `linkingObjectsOfClass:forProperty:` to `RLMObjectUtil` or elsewhere. Those are the only remaining public members of `RLMObjectBase` which shouldn't be exposed in `RealmSwift.Object`'s public interface.

I don't think the others declarations hurt us in any way. We can wait until after this PR too.